### PR TITLE
UI Bump after repo rename (CI CHANGES)

### PR DESCRIPTION
**Pull request by juliangieseke**
**What type of PR is this?**

**What this PR does/ why we need it**:

First bump after renaming the UI repo - some CI changes were necessary - 🤞 that I did it correct.

**Which issue(s) this PR fixes**:

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
Bump UI to 6.67.0
```

**Checklist**

* [ ] *If a chart is changed, the chart version is correctly incremented.*
* [ ] The commit message explains the changes and why are needed.
* [ ] The code builds and passes lint/style checks locally.
* [ ] The relevant subset of integration tests pass locally.
* [ ] The core changes are covered by tests.
* [ ] The documentation is updated where needed.

### CODE REVIEW

**Review by DanielMSchmidt — APPROVED** [review]

**Comment by juliangieseke** [comment]
```
2020-12-10T15:11:43.567521975Z Preparing worktree (detached HEAD 7b64816e)
2020-12-10T15:11:43.686963963Z HEAD is now at 7b64816e Fix cron condition (#924)
2020-12-10T15:11:43.686996156Z >>> helm dependency build ct_previous_revision042794269/stable/opsportal
2020-12-10T15:11:43.742651532Z manager.go:107: warning: a valid Helm v3 hash was not found. Checking against Helm v2 hash...
2020-12-10T15:11:43.744166243Z Error: no repository definition for https://mesosphere.github.io/kommander/charts. Please add the missing repos via 'helm repo add'
2020-12-10T15:11:43.746644120Z Error building dependencies for previous revision of chart 'opsportal => (version: "0.3.25", path: "stable/opsportal")'
2020-12-10T15:11:43.746661773Z : Error waiting for process: exit status 1
2020-12-10T15:11:43.746667088Z >>> helm dependency build stable/opsportal
```

hm, not sure what exactly thats trying to tell me.

File: stable/kommander/requirements.lock [hunk 1]
```diff
@@ -4,18 +4,18 @@ dependencies:
   version: 0.1.15
 - name: kommander-federation
   repository: https://mesosphere.github.io/yakcl/charts
-  version: 0.6.9
+  version: v0.6.9
 - name: kommander-licensing
   repository: https://mesosphere.github.io/yakcl/charts
-  version: 0.6.9
+  version: v0.6.9
```
Comment (s12chung) [inline stable/kommander/requirements.lock:10]:
```suggestion
  version: 0.6.9
```

don't think this change is needed

File: stable/kommander/requirements.lock [hunk 2]
```diff
@@ -4,18 +4,18 @@ dependencies:
   version: 0.1.15
 - name: kommander-federation
   repository: https://mesosphere.github.io/yakcl/charts
-  version: 0.6.9
+  version: v0.6.9
```
Comment (s12chung) [inline stable/kommander/requirements.lock:7]:
```suggestion
  version: 0.6.9
```

don't think this change is needed

File: stable/kommander/requirements.lock (hunk 1 above)
Comment (juliangieseke) [reply stable/kommander/requirements.lock:10]: its not, but `hlem dep up` sometimes does it… we dont really know why it happens, but its not part of the problem.

File: stable/kommander/requirements.lock (hunk 2 above)
Comment (juliangieseke) [reply stable/kommander/requirements.lock:7]: see above

File: stable/kommander/requirements.lock (hunk 1 above)
Comment (s12chung) [reply stable/kommander/requirements.lock:10]: yeah, not part of the problem.

**Comment by juliangieseke** [comment]
```2020-12-10T15:13:13.803519130Z Error: requirements.lock is out of sync with requirements.yaml```

missed that in the first time… IIRC thats because I used helmv3 locally :disappointed: is there any way to fix helmv2 CI for a local helmv3? switching is kinda tedious

**Comment by juliangieseke** [comment]
CI is green, need one more ✅ 🙇

**Pull request merged**